wallet/src/main/resources/application.conf

```diff
@@ -0,0 +1,40 @@
+commonNodeDbSettings = ${commonDbSettings}
+commonNodeDbSettings {
+  dbName = "walletdb.sqlite"
+  db {
+    user = "walletdb"
+  }
+}
+
+mainnetDb = ${commonNodeDbSettings}
+mainnetDb {
+  dbPath = ${HOME}"/.bitcoin-s/mainnet/"
+  db {
+    url="jdbc:sqlite:"${mainnetDb.dbPath}${mainnetDb.dbName}
+  }
+}
+
+testnet3Db = ${commonNodeDbSettings}
+testnet3Db {
+  dbPath = ${HOME}"/.bitcoin-s/testnet3/"
+  db {
+    url="jdbc:sqlite:"${testnet3Db.dbPath}${testnet3Db.dbName}
+  }
+}
+regtestDb = ${commonNodeDbSettings}
+regtestDb {
+  dbPath = ${HOME}"/.bitcoin-s/regtest/"
+  db {
+    url="jdbc:sqlite:"${regtestDb.dbPath}${regtestDb.dbName}
+  }
+}
+
+unittestDb = ${commonNodeDbSettings}
+unittestDb {
+  dbPath = ${HOME}"/.bitcoin-s/unittest/"
+  db {
+    url="jdbc:sqlite:"${unittestDb.dbPath}${unittestDb.dbName}
```

Comment:
Let's consider changing this to an in memory sqlite database like i did in https://github.com/bitcoin-s/bitcoin-s-core/blob/node/chain/src/main/resources/reference.conf#L37